src/components/NotificationSkeleton.tsx

```
@@ -0,0 +1,75 @@
+    <View className="px-4 py-3 border-b border-gray-800">
+      <View className="flex-row items-start">
+        {/* Avatar skeleton */}
+        <Animated.View 
+          style={shimmerStyle}
+          className="w-10 h-10 bg-gray-700 rounded-full mr-3"
+        />
+        
+        <View className="flex-1">
+          {/* Title skeleton */}
+          <Animated.View 
+            style={shimmerStyle}
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)